includes/subscriptions/class-wc-payments-subscription-migration-log-handler.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Class WC_Payments_Migration_Log_Handler
+ *
+ * @package WooCommerce\Payments
+ */
+
+use Automattic\Jetpack\Constants;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class for handling Stripe billing -> tokenized migration logs.
+ */
+class WC_Payments_Subscription_Migration_Log_Handler {
+
+	/**
+	 * Log handle (name).
+	 *
+	 * @var string
+	 */
+	const HANDLE = 'woo-payments-subscription-migration';
```

Comment:
```suggestion
	const HANDLE = 'woopayments-subscription-migration';
```

Small nit but since there's no space in WooPayments I don't think this needs to be separated by a dash.

Reply:
👍